File: Java/Beginner/1013/Main.java
import java.util.Scanner;

class Main {

    public static void main(String args[]) {
        Scanner scan = new Scanner(System.in);

        int a = scan.nextInt(), b = scan.nextInt(), c = scan.nextInt(), greatest;

        greatest = (a+b + Math.abs(a-b))/2;
        greatest = (greatest + c + Math.abs(greatest-c))/2;

        System.out.println(greatest+" eh o maior");

        scan.close();
    }

}

File: C/Beginner/1013/main.c
#include <stdio.h>
 
int main() {
 
    int a, b, c;
    int maior;
    scanf("%d%d%d", &a, &b, &c);
    
    if(b > a && b > c) maior = b;
    else if(c > a && c > b) maior = c;
    else maior = a;

    printf("%d eh o maior\n", maior);
 
    return 0;
}